src/net/http.rs

```diff
@@ -129,6 +131,11 @@ lazy_static! {
         *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX
     ))
     .unwrap();
+    static ref PATH_GET_IS_TRAIT_IMPLEMENTED: Regex = Regex::new(&format!(
+        "^/v2/traits/(?P<address>{})/(?P<contract>{})/(?P<traitName>{})/(?P<traitContractAddr>{})/(?P<traitContractName>{})$",
```

Comment:
For traits, they're usually reference like `<traitContractAddr>.<traitContractName>.<traitName>`, so it'd be more consistent for this path to match that order as well.